### DISCUSSION TITLE
Proposal to drop https://github.com/PartialVolume/shredos.i686

### DISCUSSION THREAD

**Discussion by PartialVolume** [PartialVolume/shredos.x86_64#88] [Ideas]
I'm considering dropping the [https://github.com/PartialVolume/shredos.i686/](https://github.com/PartialVolume/shredos.i686/) repository and placing the .img's and .iso's for different architectures under the [https://github.com/PartialVolume/shredos.x86_64/](https://github.com/PartialVolume/shredos.X86_64/)

I will still produce 32 bit and 64 bit and possibly add 486 and Sun Sparc Ultra builds but put the .img/.iso under the x86_64 repository.

Here's the pros

- Less time consuming for me to maintain a single shredos repository
- The differences between the x86_64 and i586/686 architectures and .img/.iso is literally only changes in 'make menuconfig', that you can count on one hand.

Here's the cons

- I can determine the total number of downloads for the x86_64 repository currently standing at 21,000 and also downloads for specific assets in a release but can't display the running total of x86_64 vs i586 as is currently displayed. Does that even matter ?

Maybe it would be better for me to put the specific release asset download counters in the release notes rather than the README.md. Maybe that would make more sense.

Any comments and suggestions are always very welcome.

**Comment by Firminator** [comment]
I'd say go for it. Anything that reduces complexity and time to compile.
